### PR TITLE
[registrypackages] Install flannel 0.8.7 binary to CentOS due to flannel binary missing in kubernetes-cni 1.1.1

### PR DESCRIPTION
**Pull request by gmaiorov**
Signed-off-by: Gleb Maiorov <gleb.maiorov@flant.com>

## Description

Install `flannel` 0.8.7 binary to CentOS due to flannel binary missing in `kubernetes-cni` 1.1.1.
Flannel was moved to sepatate repo in https://github.com/containernetworking/plugins/pull/633, so this PR is intended as temporary fix.

Changes were tested manually in paused e2e cluster.

## Why do we need it, and what problem does it solve?

Update to K8s 1.23 broken on CentOS-based systems due to this issue.
## What is the expected result?

Update to K8s 1.23 works on CentOS-based systems.
## Checklist
- [ ] The code is covered by unit tests.
- [x] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [x] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: registrypackages
type: fix
summary: Install flannel 0.8.7 binary to CentOS due to flannel binary missing in kubernetes-cni 1.1.1
impact_level: default
```

### CODE REVIEW

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @gmaiorov. I've started the workflow for label 'e2e/run/aws'...

**Comment by gmaiorov** [comment]
/e2e/abort aws;WithoutNAT;containerd;1.23 3891508847 3891508847-1-con-1-23 /sys/deckhouse-oss/install:pr3503 centos@35.156.149.189

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @gmaiorov. I've started the workflow for label 'e2e/run/aws'...

**Comment by gmaiorov** [comment]
/e2e/abort aws;WithoutNAT;containerd;1.23 3891508847 3891508847-1-con-1-23 /sys/deckhouse-oss/install:pr3503 centos@35.156.149.189

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @gmaiorov. I've recognized your '/e2e/abort' command and started the workflow...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @gmaiorov. I've started the workflow for label 'e2e/run/aws'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @gmaiorov. I've started the workflow for label 'e2e/run/aws'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @gmaiorov. I've started the workflow for label 'e2e/run/openstack'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @gmaiorov. I've started the workflow for label 'e2e/run/vsphere'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @gmaiorov. I've started the workflow for label 'e2e/run/static'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @gmaiorov. I've started the workflow for label 'e2e/run/vsphere'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @gmaiorov. I've started the workflow for label 'e2e/run/static'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @gmaiorov. I've started the workflow for label 'e2e/run/openstack'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @gmaiorov. I've started the workflow for label 'e2e/run/aws'...

**Comment by gmaiorov** [comment]
/e2e/abort aws;WithoutNAT;containerd;1.21 3902365105 3902365105-1-con-1-21 /sys/deckhouse-oss/install:pr3503 centos@3.123.56.131

**Comment by gmaiorov** [comment]
/e2e/abort aws;WithoutNAT;containerd;1.23 3902365105 3902365105-1-con-1-23 /sys/deckhouse-oss/install:pr3503 centos@3.126.122.136

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @gmaiorov. I've recognized your '/e2e/abort' command and started the workflow...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @gmaiorov. I've recognized your '/e2e/abort' command and started the workflow...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @gmaiorov. I've started the workflow for label 'e2e/run/aws'...

**Comment by gmaiorov** [comment]
/e2e/abort aws;WithoutNAT;containerd;1.21 3904640148 3904640148-1-con-1-21 /sys/deckhouse-oss/install:pr3503 centos@18.184.237.9

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @gmaiorov. I've recognized your '/e2e/abort' command and started the workflow...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @gmaiorov. I've started the workflow for label 'e2e/run/aws'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @gmaiorov. I've started the workflow for label 'e2e/run/aws'...

File: modules/007-registrypackages/images/kubernetes-cni-centos/scripts/install [hunk 1]
```diff
@@ -24,5 +24,6 @@ if ! rpm --quiet -q "${rpm_name}-${rpm_version}"; then
 else
   echo "RPM ${package_file} already installed."
 fi
+[[ -e flannel ]] && cp flannel /opt/cni/bin/flannel
```
Comment (RomanenkoDenys) [inline modules/007-registrypackages/images/kubernetes-cni-centos/scripts/install:27]:
```suggestion
[[ -e flannel ]] && cp -f flannel /opt/cni/bin/flannel
```

**Review by RomanenkoDenys — CHANGES_REQUESTED** [review]
LGTM

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @gmaiorov. I've started the workflow for label 'e2e/run/aws'...

**Review by RomanenkoDenys — APPROVED** [review]

**Comment by deckhouse-BOaTswain** [comment]
Cherry pick PR [3534](https://github.com/deckhouse/deckhouse/pull/3534) to the branch [release-1.42](https://github.com/deckhouse/deckhouse/tree/release-1.42) successful!

**Pull request merged**